site/contents/write_ups/2022/04-05-22_writeup.md

```diff
@@ -0,0 +1,97 @@
+# Data Ethics Club: [Economies of Virtue: The Circulation of ‘Ethics’ in Big Tech](https://www.tandfonline.com/doi/abs/10.1080/09505431.2021.1990875?journalCode=csac20)
+<!--Please don't edit the info panel below-->
+
+
+```{admonition} What's this? 
+This is summary of Wednesday 5th May's Data Ethics Club discussion, where we discussed the paper [Economies of Virtue: The Circulation of ‘Ethics’ in Big Tech](https://www.tandfonline.com/doi/abs/10.1080/09505431.2021.1990875?journalCode=csac20) by Jake Goldenfein, Monique Mann and Declan Kuch.
+
+The summary was written by Huw Day, who tried to synthesise everyone's contributions to this document and the discussion. "We" = "someone at Data Ethics Club". 
+Nina Di Cara and Natalie Thurlby helped with the final edit.
+
+```
+
+```{image} images/EthicsAsCurrency.png
+:width: 200px
+:align: center
+:alt: UNO Meme, "Actually be Ethical Instead of just Pretending or Draw 25" with Tech Bros holding a handful of UNO cards.
+```
+
+## Introduction
+
+This week at Data Ethics Club we discussed [Economies of Virtue: The Circulation of ‘Ethics’ in Big Tech](https://www.tandfonline.com/doi/abs/10.1080/09505431.2021.1990875?journalCode=csac20), a paper by by Jake Goldenfein, Monique Mann and Declan Kuch. (This paper may be difficult to access open source without a university login).
+
+In this paper, the authors discusses some of the ethics problems in Big Tech and "how Big Tech has transformed ethics into a form of capital — a transactional
+object external to the organisation, one of the many ‘things’ contemporary capitalists must tame and procure".
+
+We talked about the difficulities in conducting ethical research that is funded by the body you are investigating, how this research is best able to bring about change and whether interventions that serve a captialistic interest always potentially harmful.
+
+## What difficulties appear when being an ethics AI researcher? 
+
+One of our members, works on training PhD students in responsible innovation for AI. We discussed the tensions between trying to train responsible AI - it's a mindset and not a checklist. There are inherent risks in framing it otherwise - what is ethical is influenced by political events, cultural beliefs, so many things and is not just a clipboard to run through at the end of a project.
```

Comment:
```suggestion
One of the attendees works on training PhD students in responsible innovation for AI. 
We discussed the tensions of trying to train responsible AI - it's a mindset and not a checklist. There are inherent risks in framing it otherwise - what is ethical is influenced by political events, cultural beliefs, so many things and is not just a clipboard to run through at the end of a project.
```